frontend/.gitignore

```diff
@@ -21,3 +21,10 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+# my additions
```

Comment:
I don't think it's a good wording, because later one will have to dig into the git history to understand whos additions this is. This comment is redundant.

Reply:
OK, I will remove this heading.